app/scripts/controllers/newfromtemplate.js

```diff
@@ -31,8 +31,7 @@ angular.module('openshiftConsole')
     keyValueEditorUtils,
     Constants) {
 
-
-    var name = $routeParams.name;
+    var name = $routeParams.template;
```

Comment:
Need to update at least Navigate service as well since we've changed a query parameter name.